src/arrayMethodSort.js

```diff
@@ -5,7 +5,41 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    let callback = compareFunction;
+
+    if (compareFunction === undefined) {
+      callback = (x, y) => {
+        const stringX = String(x);
+        const stringY = String(y);
+
+        if (stringX > stringY) {
+          return 1;
+        } else if (stringX === stringY) {
+          return 0;
+        } else {
+          return -1;
+        };
+      };
+    };
```

Comment:
define this function outside of apply `[].__proto__.sort2`, and pass as default parameter on 7th line

Reply:
Done